src/plugins/jma/datasource.py

```
@@ -0,0 +1,364 @@
+        return self._session
+        
+    async def close(self) -> None:
+        if self._session and not self._session.closed:
+            await self._session.close()
+            
+    async def get_locations(
+        self, 
+        country: Optional[str] = None,
+        limit: Optional[int] = None,
+        **filters: Any
+    ) -> List[Location]:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)